packages/appium/test/fixtures/flattened-schema.js

```diff
@@ -522,7 +522,7 @@ export default [
     },
     schema: {
       appiumCliDescription:
-        'A list of drivers to activate. Can be a comma-delimited string or path to CSV file. By default, all installed drivers will be activated.',
+        'A list of drivers to activate. Can be a comma-delimited string or path to CSV file. By default, all installed drivers will be activated. Windows environments may require wrapping the comma-delimited string with quotes to escape the comma.',
```

Comment:
i feel like these changes came in erroneously from another PR?

Reply:
or oh wait this is probably generated from that stuff.

Reply:
no, neither; this file needed updating manually as per my comment above in the issue

Reply:
👍